applications/asset_tracker_v2/doc/debug_module.rst

```diff
@@ -22,14 +22,7 @@ This section documents the various features implemented by the module.
 Memfault
 ========
 
-The debug module uses `Memfault SDK`_ to track |NCS| specific metrics such as LTE and stack metrics.
-In addition, the following types of custom Memfault metrics are defined and tracked when compiling in the debug module:
-
- * ``gnss_time_to_fix_ms`` - Time duration between the start of a GNSS search and obtaining a fix.
- * ``gnss_satellites_tracked_count`` - Number of satellites tracked during a GNSS search window.
- * ``location_timeout_search_time_ms`` - Time duration between the start of a location search and a search timeout.
-
-The debug module also implements `Memfault SDK`_ software watchdog, which is designed to trigger an assert before an actual watchdog timeout.
```

Comment:
It may be worth moving these metric descriptions into the memfault_ncs doc source, although I'm not sure that the other metrics are defined there?